es6-modules/src/lib/index.js

```diff
@@ -0,0 +1,9 @@
+import { noop } from './noop';
+import { toArray } from './to-array';
+import { createElement } from './create-element';
```

Comment:
After updating to `export default` you will need to update the `import` statements